Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -527,6 +580,21 @@ private bool TryAddMsBuildArguments(ProjectWithPredictions<AbsolutePath> project
                     project.FullPath.GetName(m_context.PathTable).ToString(m_context.StringTable));
             }
 
+            // Pass the output result cache file if present
+            if (outputResultCacheFile != AbsolutePath.Invalid)
+            {
+                using (pipDataBuilder.StartFragment(PipDataFragmentEscaping.NoEscaping, string.Empty))
+                {
+                    pipDataBuilder.Add(PipDataAtom.FromString("/orc:"));

```

Comment:
same - document /orc

Reply:
Or use long name.

Reply:
I know it probably doesn't move the perf needle, but I was intendedly using the short version to not unnecessarily bloat the response file (for a big repo, we may have many of these entries).
If this is about documenting this better, I'd prefer to just add a comment explaining what this is 

---
In reply to: [272271223](https://github.com/Microsoft/BuildXL/pull/80#discussion_r272271223) [](ancestors = 272271223)